### ISSUE TITLE
Why are you not including express?

### ISSUE THREAD

**Issue by daniel-z** [pencilblue/pencilblue#619]
I think that will help to include plugins from the express community. But I don't understand the technical implications of this.

**Comment by blakecallens** [comment]
I'll give you a security example for why we don't use 3rd party tools for the core functionality of PencilBlue, although I'm sure @brianhyder has some more reasons he can share.

Once you're in with something like Express, because it runs the base of an application, you are completely locked in to that platform. On top of that, many of the plugins for Express that provide essential functionality are built and maintained by individual developers outside of StrongLoop.

If one of those plugins, which could be essential for the running of an Express based, fully featured CMS, happened to have a major security flaw, every installation of that CMS would have a major security flaw.

If we found ourselves in that situation, and the developer wasn't active or had completely abandoned the project, what would we do? We could petition him/her to make a fix and sit around while PB was totally insecure. We could fork the repo and now have to be in charge of an Express plugin. We could completely back out of the plugin and have to write our own functionality from scratch.

Meanwhile, every single instance of PB would be insecure.

Placing the very foundation of a platform of PB's size completely in the hands of a 3rd party isn't maintainable - not to mention modifiable, should we want to do something different than StrongLoop.

Hopefully that clears up why we made this decision.
